requirements/requirements.txt

```diff
@@ -3,15 +3,15 @@ Cython==0.17.4
 restkit==4.2.0
 jsonobject-couchdbkit==0.6.5.5
 jsonobject==0.4.3
-celery==3.0.24
+celery==3.1.11
 decorator==3.3.2
 django==1.5.6
 django-braces==1.3.1
-django-celery==3.0.23
+django-celery==3.1.10
```

Comment:
Yeah but it seemed like we still might need it for certain features. I can double-check.